lib/rouge/lexers/turtle.rb

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class Turtle < RegexLexer
+      title "Turtle/TriG"
+      desc "Terse RDF Triple Language, TriG"
+      tag 'turtle'
+      filenames *%w(*.ttl *.trig)
+      mimetypes *%w(
+        text/turtle
+        application/trig
+      )
+
+      def self.analyze_text(text)
+        start = text[0..1000]
+        return 0.9 if start =~ %r(@prefix\b)
```

Comment:
please don't return more than 0.5 for strings that could appear in other languages.